moments/Parsing.py

```diff
@@ -0,0 +1,1119 @@
+        checked against the expected type for a field- e.g. if a string is 
+        given for the numeric field 'GQ', no explicit warning is raised- 
+        although an error will tend to be thrown once parsing begins.
+    'SAMPLE/FIELD' e.g. 'SAMPLE/GQ' imposes filters at the sample level. 
+        'FIELD' should correspond to an entry in the ``FORMAT`` column of the 
+        VCF. Typing is the same as in 'FORMAT'.
+    In general, when quantities are absent in a given line, the line is not 
+    skipped but a one-time alert message is raised. Missing values ('.') also
+    do not cause lines to be skipped. 
+
+    :param vcf_file: Pathname of the VCF file to parse. The file may be 
+        gzipped/bgzipped or uncompressed.
+    :type vcf_file: str
+    :param pop_file: Pathname of a whitespace-separated file mapping samples
+        to populations with the format SAMPLE POPULATION. Sample names must
+        be unique and there should be one of them on each line (default None 
+        combines all samples into a single population 'ALL'). Samples not 
+        listed here are ignored.
+    :type pop_file: str, optional
+    :param pop_mapping: Optional dictionary (default None) mapping population
+        IDs to lists of VCF sample IDs. Mutually exclusive with `pop_file`.
+    :type pop_mapping: dict, optional
+    :param bed_file: Pathname of a BED file defining the regions within which 
+        to parse; useful for applying masks to exclude difficult-to-call or 
+        functionally constrained regions (default None). 
+    :type bed_file: str, optional
+    :param interval: 2-tuple or 2-list specifying the (inclusive, 0-indexed) 
+        first and (exclusive, 0-indexed) last positions to parse; defines a
+        genomic window (default None parses the whole VCF file).
+    :type interval: tuple of integers, optional
+    :param anc_seq_file: Pathname of a FASTA file defining estimated ancestral 
+        nucleotide states (default None).
+    :type anc_seq_file: str, optional
+    :param allow_low_confidence: If True (default False) and `anc_seq_file` is
+        given, allows low-confidence ancestral state assignments- represented 
+        by lower-case nucleotide codes- to stand. If False, sites with them
+        are skipped.
+    :type skip_low_conf: bool, optional
+    :param use_AA: If True, use entries in the VCF field ``INFO/AA`` to polarize
+        alleles (default False). 
+    :type use_AA: bool, optional
+    :param filters: A dictionary mapping VCF fields to filter criteria, for
+        imposing quantitative thresholds on measures of genotype quality and 
+        subsetting to qualitative classes of sites. Filtering is discussed
+        above.
+    :type filters: dict, optional
+    :param allow_multiallelic: If True (default False), includes sites with 
+        more than one alternate allele, counting each derived allele separately-
+        otherwise skips these sites. Also allows sites those biallelic sites 
```

Comment:
Small typo. Also, can you expand briefly on the behavior of how multi-allelics contribute to the SFS, if this argument is True?